src/v2/server.ts

```diff
@@ -8,6 +8,20 @@ import type { NextFunction } from "express"
 const app = express()
 const { routes, routePaths } = getRouteConfig()
 
+// Experiment with overwriting segment consent cookie with server-side version
+app.get("/set-tracking-preferences", (req, res, next) => {
```

Comment:
@dzucconi - note that this stuff has already been merged to `main` and is no longer here: https://github.com/artsy/force/blob/main/src/server.ts#L54

Reply:
oh lol i already mentioned this